packages/loot-core/src/client/platform.js

```diff
@@ -11,7 +11,7 @@ const isProbablySafari = /^((?!chrome|android).)*safari/i.test(
 export default {
   OS: isWindows ? 'windows' : isMac ? 'mac' : 'linux',
   env: 'web',
-  isBrowser: !!window.Actual.IS_FAKE_WEB,
+  isBrowser: !!window.Actual?.IS_FAKE_WEB,
```

Comment:
When would `window.Actual` not be defined? That feels like it would cause more problems than just this one.

Reply:
uhhhhh that's a good question

Reply:
We have 3x environments now:

- browser
- jest
- electron

In Electron we want to turn off some features. This var helps with that. But perhaps there is some work we could do to move it to `NODE_ENV`? Or perhaps a different env variable that's set by electron. Definitely something we can explore more.

Currently `Actual` can be undefined in unit tests. Since I'm enabling one new test (the TransactionTable one) - the var is not set for it.

Reply:
For unit tests - definitely no. Unit tests (as the name implies) test a unit of logic in isolation. So without all the fluff of actual backend etc.

For integration tests - maybe. Ideally I would prefer to keep them as simple as possible with no unnecessary dependencies (which includes `Actual`). But maybe that won't be possible.. either way - we can cross that bridge when we get there. Currently `desktop-client` has only 1x integration test.